docs/README.md

```diff
@@ -1,16 +1,18 @@
 # Documentation
 
-- [Tutorials](/docs/tutorials.md)
-- [Contributing](/docs/CONTRIBUTING.md)
+- [Tutorials](tutorials.md)
+- [Contributing](CONTRIBUTING.md)
 - Getting Started
   - [Hello World](#hello-world)
   - [Installation](#installation)
 - Concepts
-  - [Keys](/docs/keys.md)
-  - [Lifecycle](/docs/lifecycle.md)
-  - [Components](/docs/components.md)
-  - [Sanitation](/docs/sanitation.md)
-  - [Hydration](/docs/hydration.md)
+  - [State Slices](state-slices.md)
```

Comment:
Let's call it just "Slices"? Since it includes actions as well.

Reply:
Renamed.